Dfe.Data.SearchPrototype/Web/Tests/PageObjectModel/PrivacyPage.cs

```diff
@@ -1,15 +1,16 @@
-﻿using DfE.Data.SearchPrototype.Web.Tests.Integration.PageObjectModel.Setup;
+﻿using Dfe.Data.SearchPrototype.Web.Tests.PageObjectModel.Setup;
 using Microsoft.AspNetCore.Mvc.Testing;
 
-namespace DfE.Data.SearchPrototype.Web.Tests.Integration.PageObjectModel
+namespace Dfe.Data.SearchPrototype.Web.Tests.PageObjectModel
```

Comment:
why the namespace change?

Reply:
I've moved the PageObjectModel directory up from the `Integration` dir to the root of the project.